pokete.py

```diff
@@ -1850,73 +1850,13 @@ def recogniser():
 #########################################
 
 # maps
+for ob_map in maps:
+    args = maps[ob_map]
+    args["extra_actions"] = eval(args["extra_actions"]) if args["extra_actions"] is not None else None
+    exec(f'{ob_map} = PlayMap(name = ob_map, **args)')
+
 centermap = PlayMap(height-1, width, name = "centermap", pretty_name = "Pokete-Center")
 shopmap = PlayMap(height-1, width, name = "shopmap", pretty_name = "Pokete-Shop")
```

Comment:
Shouldn't these maps also be outsourced?

Reply:
height and width are global variables only present in pokete.py, therefore those two maps cant be moved over to maps.py without any additional work.

Reply:
Ok nice.

Maybe add a comment explaining this?

Reply:
centermap and shopmap don't have fixed sizes, their size is the windows size it self..

Reply:
Ok. Was taking about

```python
#centermap and showmap cannot be outsourced because of their dynamic sizes
```

but thanks for the advanced explanation! 😂

Reply:
I didn't see your comment as I was typing the second reply, and I added a comment. :laughing: